Makefile

```diff
@@ -7,3 +7,9 @@ run:
 
 run-nohup:
 	nohup Rscript run.R &
+
+sync:
+	Rscript sync.R
+
+dashboard:
```

Comment:
Consider using the `dashboard` make target as a dependency within make, rather than doing `source("dashboard.R")` from `run.R`.